core/domain/suggestion_services.py

```diff
@@ -4001,8 +4001,7 @@ def _generate_translation_contributor_certificate_data(
     hours_contributed = round(words_count / 300, 2)
 
     if words_count == 0:
-        raise Exception(
-            'There are no contributions for the given time range.')
+        return None
```

Comment:
Could you please update the backend tests for this change. I think backend tests in the CI are failing due to this.

Reply:
Got it! suggestion_services.py has had its tests updated.

Reply:
Hi @chris7716, would you mind taking a look at this again?